pkg/apis/operator/v1alpha1/tektonchain_validation.go

```diff
@@ -25,7 +25,12 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var allowedArtifactsStorage = sets.NewString("tekton", "oci", "gcs", "docdb", "grafeas", "kafka")
+var (
+	allowedArtifactsTaskRunFormat    = sets.NewString("in-toto", "slsa/v1")
+	allowedArtifactsStorage          = sets.NewString("tekton", "oci", "gcs", "docdb", "grafeas", "kafka")
+	allowedX509SignerFulcioProvider  = sets.NewString("google", "spiffe", "github", "filesystem")
+	allowedTransparencyConfigEnabled = sets.NewString("true", "false", "manual")
```

Comment:
Seems all these sets allowed empty string, can we add empty string in the allowed sets and omit `if xyz != ""` statement on the validation?